### PR TITLE
create stacktrace

### PR DESCRIPTION
**Pull request by szuecs**
print proper stacktrace in our recover handler, if we catch the panic

### CODE REVIEW

File: worker.go [hunk 1]
```diff
@@ -22,7 +23,7 @@ func startPolling(awsAdapter *aws.Adapter, kubeAdapter *kubernetes.Adapter, poll
 func doWork(awsAdapter *aws.Adapter, kubeAdapter *kubernetes.Adapter) error {
 	defer func() error {
 		if r := recover(); r != nil {
-			log.Println("shit has hit the fan:", r)
+			log.Println("shit has hit the fan:", errors.Wrap(r.(error), "panic caused by"))
```
Comment (mikkeloscar) [inline worker.go]: Does this actually show the stacktrace where the error occurred? I would assume it's just the stack of this local recover function, but maybe I'm wrong?

Comment (szuecs) [inline worker.go]: I honestly don't know, but the hope is to get the real stacktrace from runtime.Callers.

**Comment by mikkeloscar** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**